control/tests/convert_test.py

```diff
@@ -184,6 +184,28 @@ def testConvertMIMO(self):
         if (not slycot_check()):
             self.assertRaises(TypeError, control.tf2ss, tfcn)
 
+    def testTf2ssStaticSiso(self):
+        """Regression: tf2ss for SISO static gain"""
+        import control
+        gsiso = control.tf2ss(control.tf(23, 46))
+        self.assertEqual(0, gsiso.states)
+        self.assertEqual(1, gsiso.inputs)
+        self.assertEqual(1, gsiso.outputs)
+        # in all cases ratios are exactly representable, so assert_array_equal is fine
+        np.testing.assert_array_equal([[0.5]], gsiso.D)
+
+    def testTf2ssStaticMimo(self):
+        """Regression: tf2ss for MIMO static gain"""
+        import control
+        # 2x3 TFM
+        gmimo = control.tf2ss(control.tf([[ [23],   [3],  [5] ], [ [-1],  [0.125],  [101.3] ]],
```

Comment:
It's an arbitrary selection of numbers over a (small) range of magnitudes and signs; in all cases the ratios are exactly representable with low precision (see d just below), so I could use np.assert_array_equal().